elasticsearch/run.sh

```diff
@@ -60,4 +57,18 @@ else
 	exit 1
 fi
 
-exec /usr/share/elasticsearch/bin/elasticsearch
+TIMES=60
```

Comment:
basically -- we try to connect to ES for 60 seconds until we see that the searchguard index has been created. ES will start allowing connections through once it finishes loading in indices, so if it doesn't start up within 60 seconds then we bail.

Think this should be bumped up from 60?